lib/v2/screens/explore_screens/vote_screens/proposal_details/viewmodels/proposal_details_event.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+
+/// --- EVENTS
+@immutable
+abstract class ProposalDetailsEvent extends Equatable {
+  const ProposalDetailsEvent();
+  @override
+  List<Object> get props => [];
+}
+
+class OnFavourButtonTapped extends ProposalDetailsEvent {
```

Comment:
Is this for the Yes button?

Reply:
Yes, for now i just add the nessesary logic to show this fuctionality.